src/components/Spawn/Layout.tsx

```diff
@@ -22,18 +23,25 @@ export const StyledBadge = styled(Badge)`
   margin: 0 ${size.xs};
 `;
 
-export const PlusButton = ({ children, ...props }) => (
-  <Button {...{ ...props, glyph: <Icon glyph="Plus" /> }} as="button">
-    {children}
-  </Button>
-);
+export const PlusButton: ExtendableBox<
```

Comment:
This button was never actually typed correctly so it broke when we updated the Leafygreen Button and the props changed.